documentation/proc-pages/development/standards.md

```diff
@@ -1,11 +1,17 @@
 # Style Guide
 
+`PROCESS` follows the Python [PEP8](https://peps.python.org/pep-0008/) style guide for its layout and namespace style. This should not conflict with Fortran90 in terms of names but may with the line lengths.
```

Comment:
Could you state what the line length limit is, and maybe provide an example?

Reply:
This is what I wanted to know about for the Fortran and how we may be doing it differently.
For PEP 8 it is:

"Limit all lines to a maximum of 79 characters."

"For flowing long blocks of text with fewer structural restrictions (docstrings or comments), the line length should be limited to 72 characters."

Reply:
I believe there is a line length limit in Fortran, but it's bigger than the PEP8 recommendations. I'd just include what you've commented above: this will probably be enforced by a formatter at some point anyway.